lib/fog/aws/requests/dynamodb/query.rb

```diff
@@ -25,15 +23,17 @@ class Real
         #     * 'Count'<~Integer> - number of items in response
         #     * 'Items'<~Array> - array of items returned
         #     * 'LastEvaluatedKey'<~Hash> - last key scanned, can be passed to ExclusiveStartKey for pagination
-        def query(table_name, hash_key, options = {})
+        #
+        # See DynamoDB Documentation: http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html
+        #
+        def query(table_name, options = {})
```

Comment:
I've updated this to show a deprecation notice and fallback to the former api version.